elliottlib/runtime.py

```diff
@@ -276,15 +277,15 @@ def image_metas(self):
     def rpm_metas(self):
         return list(self.rpm_map.values())
 
-    @property
-    def bug_trackers(self):
-        if self._bug_trackers:
-            return self._bug_trackers
-        if not self.only_jira:
-            self._bug_trackers['bugzilla'] = BugzillaBugTracker(BugzillaBugTracker.get_config(self))
-        if self.use_jira or self.only_jira:
-            self._bug_trackers['jira'] = JIRABugTracker(JIRABugTracker.get_config(self))
-        return self._bug_trackers
+    def bug_trackers(self, bug_tracker_type):
```

Comment:
This was a problematic piece of logic which was too rigid
for supporting all of different command configurations. so refactor
it to give it what the command wants